### ISSUE TITLE
Chosen-like drop-downs in Advanced search inaccessible

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#304]
1. "it's a dropdown list but it says it's a button"
1. when tabbing through options inside a dropdown nothing is announced

_from accessibility testing with Michelle on 5/14/19_

Aha! Link: https://penn-state-university-libraries.aha.io/features/STPROJ-41
